common-tasks/tips-and-tricks.md

```diff
@@ -46,7 +46,7 @@ Preventing data leaks
 ---------------------
 First make sure to read [Understanding and Preventing Data Leaks](/doc/data-leaks/) section to understand the limits of this tip.
 
-Suppose that you have a not so trusted enviroment, for example a Windows VM, an application that track and report it's usage or you simply want to protect your data.
+Suppose that you have in a not so trusted enviroment, for example a Windows VM, an application that track and report its usage, or you simply want to protect your data.
```

Comment:
It's better English because the other side is 'an application'

You can't 'have a not so trusted environment, an application'

but you can have (in) a not so trusted environment, an application

The application is 'in' the environment.